src/app/shell/details/competition-details/competition-details.component.ts

```
@@ -50,6 +63,8 @@ export class CompetitionDetailsComponent extends TabParamsComponent implements O
 
   public isImageBroken: boolean = false;
   public competitionStatusOpen: boolean;
+  public isImageBroken: boolean = false;
```

Biome (2.1.2)

[error] Duplicate class member name "isImageBroken"



(lint/suspicious/noDuplicateClassMembers)